src/api/spec/controllers/webui/users/tokens_controller_spec.rb

```diff
@@ -0,0 +1,169 @@
+require 'rails_helper'
+
+RSpec.describe Webui::Users::TokensController, type: :controller do
+  let(:user) { create(:confirmed_user, login: 'foo') }
+  let(:other_user) { create(:confirmed_user, login: 'bar') }
+
+  before do
+    Flipper[:trigger_workflow].enable
+  end
+
+  describe 'GET #index' do
+    context 'user is not logged in' do
+      it_behaves_like 'require logged in user' do
```

Comment:
`require_login` is already tested in the `WebuiController` spec. No need to do it again here for every action.

Reply:
"require logged in user" tests removed.